data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Northern Blockade"
+	minor
+	name `Mebsuta Disaster`
+	description `<destination> is under attack by pirates and the Navy needs a captain to deliver <commodity> to the planet immediately.`
```

Comment:
I suggest we rewrite this in a standard mission format: instead of saying that the Navy needs a captain, straight-up tell the player that he should go to `<destination>`.
> `<destination>` is under attack by pirates. Under Navy orders, deliver `<commodity>` to the planet as soon as possible.